bots/activity.bot.js

```diff
@@ -36,7 +36,7 @@ const debug = (...args)=>DEBUG && Slack.log(args.join(' '));
  * more or less.
  */
 const tallyMessage = (msg)=>{
-	const channelIgnored = _.has(msg.channel, IGNORED_CHANNELS);
+	const channelIgnored = _.indexOf(IGNORED_CHANNELS, msg.channel) > -1;
```

Comment:
`_.includes(IGNORED_CHANNELS, msg.channel)`

Reply:
I _knew_ I was forgetting how to do this!